### ISSUE TITLE
zmb has unexpected handling of nameserver option

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#780]
For "start_domain_test" one of the options is "--nameserver DOMAIN_NAME:IP_ADDRESS". It does not say how to handle empty IP address.

Just the domain name does not work, which is the natural choice.

```sh
zmb --verbose start_domain_test --nameserver ns.nic.se --domain iis.se
{"params":{"domain":"iis.se","nameservers":[{"ip":null,"ns":"ns.nic.se"}]},"jsonrpc":"2.0","method":"start_domain_test","id":1}
{"error":{"data":"/nameservers/0/ip: Expected string - got null.","code":"-32602","message":"Invalid method parameter(s)."},"jsonrpc":"2.0","id":1}
```
You have to include the colon. It is unexpected, and should at least be doucmented.

```sh
zmb --verbose start_domain_test --nameserver ns.nic.se: --domain iis.se
{"method":"start_domain_test","params":{"nameservers":[{"ns":"ns.nic.se","ip":""}],"domain":"iis.se"},"jsonrpc":"2.0","id":1}
{"result":"9fa9d9707838f67b","id":1,"jsonrpc":"2.0"}
```

**Comment by mattias-p** [comment]
This is related to #708.

**Comment by matsduf** [comment]
A simple solution would be to change from
```
   start_domain_test
        zmb [GLOBAL OPTIONS] start_domain_test [OPTIONS]

        Options:

           --domain DOMAIN_NAME
           --ipv4 true|false|null
           --ipv6 true|false|null
           --nameserver DOMAIN_NAME:IP_ADDRESS
```
to
```
   start_domain_test
        zmb [GLOBAL OPTIONS] start_domain_test [OPTIONS]

        Options:

           --domain DOMAIN_NAME
           --ipv4 true|false|null
           --ipv6 true|false|null
           --nameserver DOMAIN_NAME:IP_ADDRESS
           --nameserver DOMAIN_NAME:  # Mandatory trailing ":" when IP address is empty
```

**Comment by matsduf** [comment]
Resolved by #785.
